zipkin-collector/kafka/src/main/java/zipkin2/collector/kafka/KafkaCollector.java

```diff
@@ -148,9 +153,12 @@ public KafkaCollector build() {
   }
 
   final LazyKafkaWorkers kafkaWorkers;
+  private final Properties properties;
+  private AdminClient adminClient;
```

Comment:
I would change this to package specific access to reduce the size of the bytecode.

Reply:
@zeagord didn't know that, thanks! :) Fixing it.